src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionSpec.scala

```diff
@@ -348,6 +348,45 @@ class GoogleExtensionSpec(_system: ActorSystem) extends TestKit(_system) with Fl
     }
   }
 
+  it should "get a group's email" in {
```

Comment:
it's not clear to me what the purpose of this unit test is or how it provides value over the test on line 377